setup_support.py

```diff
@@ -56,6 +56,15 @@ def finalize_options(self):
         self.announce('Writing man page(s) to %s' % self.manpath)
         self._today = datetime.date.today()
 
+    @property
+    def version(self):
```

Comment:
no self is used, so no need for it to be a property.  make it a regular function and use it also in setup.py instead of duplicated now code